nuit/templatetags/nuit.py

```diff
@@ -350,7 +352,10 @@ def render(self, context):
                     row_data.append((field_data, {}))
                 else:
                     try:
-                        row_data.append((field_name, literal_eval(data)))
+                        aeval = asteval.Interpreter()
```

Comment:
Arguably, Django could provide an `items` method to the `Context` object, but it doesn't 😔 it would negate the need for my for loop too.